src/renderer/src/routes/pages/Swap/SwapPageContent.tsx

```diff
@@ -277,6 +280,19 @@ export const SwapPageContent = ({ account }: TProps) => {
     swapServiceRef.current.setExtraIdToReceive(event.target.value)
   }
 
+  const handlePasteAddressToReceive = () => {
+    pressingPasteAddressToReceive.onPress(async () => {
+      try {
+        const text = await navigator.clipboard.readText()
+
+        swapServiceRef.current!.setAddressToReceive(text)
+      } catch (error) {
+        ToastHelper.success({ message: tCommonGeneral('pasteFromClipboardError') })
+        console.error(error)
+      }
+    })
+  }
```

Comment:
I think a better approach would be:

```suggestion
  const handlePasteAddressToReceive = () => {
      try {
        const text = await navigator.clipboard.readText()

        swapServiceRef.current!.setAddressToReceive(text)
      } catch (error) {
        ToastHelper.success({ message: tCommonGeneral('pasteFromClipboardError') })
        console.error(error)
      }
  }
  
  <Component onPress={handlePressOnce(handlePasteAddressToReceive)} />
```